charts/fluent-operator/values.yaml

```diff
@@ -22,9 +24,10 @@ operator:
       requests:
         cpu: 50m
         memory: 64Mi
-  container:
-    repository: "kubesphere/fluent-operator"
-    tag: "latest"
+  image:
+    registry: ghcr.io
+    repository: fluent/fluent-operator/fluent-operator
```

Comment:
Aligned the registry and repository with the other images from fluent-bit and fluentd.